git/git_test.go

```diff
@@ -66,7 +66,7 @@ func TestCurrentRefAndCurrentRemoteRef(t *testing.T) {
 
 	refname, err := RemoteRefNameForCurrentBranch()
 	assert.Equal(t, nil, err)
-	assert.Equal(t, "origin/someremotebranch", refname)
+	assert.Equal(t, "refs/remotes/origin/someremotebranch", refname)
```

Comment:
Could use this as an excuse to swap out `github.com/technoweenie/assert` for `github.com/stretchr/testify/assert`, if you're cool with that 😄